nodejs/app.js

```diff
@@ -0,0 +1,48 @@
+const knex = require('knex')({
+    client: 'mysql2',
+    connection: {
+        host: process.env.DB_HOST || '0.0.0.0',
+        port: process.env.DB_PORT || 3306,
+        user: process.env.DB_USER || 'root',
+        password: process.env.DB_PASSWORD || '1234',
```

Comment:
You probably know this already and it not so important for this assignment but .env.example file instead of clear text password. ;)

Reply:
I added now the .env.example file.

Reply:
I also removed the files in the wrong folder.

Thank you for the help!